.github/workflows/run-tests.yaml

```diff
@@ -20,20 +20,22 @@ jobs:
           - windows-latest
           - macos-latest
         python-version:
-          - '3.6'
-          - '3.7'
           - '3.8'
           - '3.9'
           - '3.10'
           - '3.11'
-          # - '3.12'  # FixMe: https://github.com/pylint-dev/pylint-pytest/issues/3
-        # Python 3.6 is not available in `ubuntu-latest`.
-        exclude:
-          - python-version: '3.6'
-            os: ubuntu-latest
+          - '3.12'
         include:
-          - python-version: '3.6'
-            os: ubuntu-20.04
+          - python-version: '3.8'
+            tox_env: py38
+          - python-version: '3.9'
+            tox_env: py39
+          - python-version: '3.10'
+            tox_env: py310
+          - python-version: '3.11'
+            tox_env: py311
+          - python-version: '3.12'
+            tox_env: py312
```

Comment:
I'd ... want to avoid that, if possible.

It was "fine" for me that Github runners being _weird_ and running every time all Python version _for now_.

Your previous suggestion of `py${matrix.tox_env//./}` would have been acceptable.
I'm inclined to say no to this 😕

Reply:
that failed on windows but i've reverted to it.

Reply:
I'll see what i can do to fix it on windows